kube-config-files/methode-article-mapper-service.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name:  methode-article-mapper
+  labels:
+    name:  methode-article-mapper
+    visualize: "true"
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: methode-article-mapper
+  template:
+    metadata:
+      labels:
+        app:  methode-article-mapper
+        visualize: "true"
+    spec:
+      containers:
+      - name:  methode-article-mapper
+        image:  coco/methode-article-mapper:rework-config
+        imagePullPolicy: Always
+        resources:
+          limits:
+            memory: 256Mi
+        livenessProbe:
+          initialDelaySeconds: 15
+          tcpSocket:
+            port: 8080
+        readinessProbe:
+          httpGet:
+            path: /__health
```

Comment:
this readinessProbe might not work as expected with health endpoint, because the health endpoint responds with 200 status code even if there is a healthcheck that is false. The readinessProbe will take action on the pod only if it gets a 5xx status code on the probe, therefore my suggestion is to temporary remove it and implement the gtg endpoint for this service when possible, then add back the readiness probe